config/environments/development.rb

```diff
@@ -9,59 +9,43 @@
   # Do not eager load code on boot.
```

Comment:
Looks like this is just everything slightly reorganized, except for `public_file_server_enabled` and `actionmailer.default_url_options` being removed.